### DISCUSSION TITLE
Build error with vcpkg

### DISCUSSION THREAD

**Discussion by chuyanjuncn** [pelicanmapping/osgearth#1834] [Support Q&A] [answered]
I am building the Ver. 3.2 with vcpkg.  There is a fatal error with imgui. 
FAILED: src/applications/osgearth_imgui/CMakeFiles/application_osgearth_imgui.dir/__/__/third_party/imgui/backends/imgui_impl_opengl3.cpp.obj 
D:\PROGRA~1\MICROS~1.0\VC\bin\amd64\cl.exe   /TP -DGEOS_INLINE -DH5_BUILT_AS_DYNAMIC_LIB -DOSGEARTH_ENABLE_IMGUI -DOSGEARTH_HAVE_MBTILES -DPROTOBUF_USE_DLLS -DUSE_UNSTABLE_GEOS_CPP_API -D_CRT_SECURE_NO_DEPRECATE -D_SCL_SECURE_NO_WARNINGS -II:\vcpkg\buildtrees\osgearth\x64-windows-dbg\build_include -II:\vcpkg\buildtrees\osgearth\src\06c3d361c0-8cb1d1bee5.clean\src -II:\vcpkg\buildtrees\osgearth\src\06c3d361c0-8cb1d1bee5.clean\src\third_party\imgui -II:\vcpkg\buildtrees\osgearth\src\06c3d361c0-8cb1d1bee5.clean\src\third_party\imgui\examples -II:\vcpkg\installed\x64-windows\include -II:\vcpkg\installed\x64-windows\include\geos -II:\vcpkg\installed\x64-windows\include\cfitsio -II:\vcpkg\installed\x64-windows\include\server -II:\vcpkg\installed\x64-windows\include\libxml2 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /fp:precise /showIncludes /Fosrc\applications\osgearth_imgui\CMakeFiles\application_osgearth_imgui.dir\__\__\third_party\imgui\backends\imgui_impl_opengl3.cpp.obj /Fdsrc\applications\osgearth_imgui\CMakeFiles\application_osgearth_imgui.dir\ /FS -c I:\vcpkg\buildtrees\osgearth\src\06c3d361c0-8cb1d1bee5.clean\src\third_party\imgui\backends\imgui_impl_opengl3.cpp
I:\vcpkg\buildtrees\osgearth\src\06c3d361c0-8cb1d1bee5.clean\src\third_party\imgui\backends\imgui_impl_opengl3.cpp(107): fatal error C1083: Cannot open include file: 'GL/gl3w.h': No such file or directory
[install-x64-windows-dbg-out.log](https://github.com/gwaldron/osgearth/files/7379280/install-x64-windows-dbg-out.log)

**Comment by plevy** [comment]
Do you have glew:x64-windows installed?  GL3W isn't supported right now.

Thank you

**Comment by chuyanjuncn** [comment]
Thanks a lot.
How to correct this error.

**Comment by jasonbeverage** [comment, marked as answer]
Like Paul said, if you are trying to build the imgui applications you need glew installed and it needs to be found by cmake.  You can do that with vcpkg install glew:x64-windows. 

That error indicates that no opengl loader was found by imgui and it's trying to default to the gl3w loader which isn't actually  included in our imgui source tree.

**Comment by chuyanjuncn** [comment]
Glew has already been installed, but the error still exists.
